### PR TITLE
Adds CI Hooks for Pull Requests

### PR DESCRIPTION
**Pull request by edgibbs**
Also added a minimal docker image with headless Chrome.

### CODE REVIEW

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
Retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM node:carbon
```
Comment (eckdanny-osi) [inline Dockerfile]: is `debian:jessie` underneath a few layers ✔︎

**Pull request merged**